js/Localization.js

```diff
@@ -0,0 +1,114 @@
+import i18n from 'i18n-js';
+import * as RNLocalize from 'react-native-localize';
+
+i18n.translations = {
+  en: {
+    add_first_site: '+ Add your first site',
+    add_site: 'Add Site',
+    add_sites: 'Add Discourse sites to keep track of.',
+    add: '+ Add',
+    all: 'All',
+    and: 'and',
+    approved_commits: '%{count} commits approved',
+    approved: '"%{title}" approved',
+    check_out_popular: 'Check out these popular communities.',
+    connect: 'connect',
+    dont_know_where_to_start: 'Don’t know where to start?',
+    inbox_message: '%{inbox_count} %{messages} in your %{group_name} inbox',
```

Comment:
This should solve your manual pluralization where you try to switch between "message" and "messages".

```suggestion
    inbox_message: {
      one: '%{count} message in your %{group_name} inbox',
      other: '%{count} messages in your %{group_name} inbox',
    }
```